pkgs/applications/display-managers/lightdm-enso-os-greeter/default.nix

```diff
@@ -0,0 +1,72 @@
+{ stdenv, fetchgit, pkgconfig
+, dbus, pcre, epoxy, libXdmcp, at-spi2-core, libxklavier, libxkbcommon, libpthreadstubs
```

Comment:
These packages are just to resolve warnings(?) raised by pkgconfig. Seems to compile fine without them.